src/loadCompilerOptions.ts

```diff
@@ -0,0 +1,20 @@
+import * as ts from "typescript";
+
+export const loadCompilerOptions = (): ts.CompilerOptions => {
+  const configFileName = ts.findConfigFile(process.cwd(), ts.sys.fileExists);
+  if (!configFileName) {
+    throw new Error("Could not find config file!");
+  }
+
+  const result = ts.readConfigFile(configFileName, ts.sys.readFile);
+  if (result.error) {
+    let message;
+    if (typeof result.error.messageText === "string") {
+      message = result.error.messageText;
+    } else {
+      message = result.error.messageText.messageText;
+    }
```

Comment:
What is this doing and could you leave a comment about it?

Reply:
According to TypeScript the type of `result.error.messageText` is `string | DiagnosticMessageChain` and `DiagnosticMessageChain` in turn has a `messageText` property. I was wondering if we should just return an empty object from this function if we can't find or read the config file. What do you think?

Reply:
Yeah I agree 👍, I feel like any real issues with the config file (such as syntax errors) will most likely be surfaced in a better way while using any other TS tooling anyways.